tests/stress/tx_stress/test_stress_txs.py

```diff
@@ -0,0 +1,58 @@
+from pytest_testconfig import config as testconfig
+import time
+import multiprocessing as mp
+
+from tests.convenience import sleep_print_backwards
+from tests.queries import get_release_tick_msgs
+from tests.test_bs import setup_network, add_curl, setup_bootstrap, start_poet, setup_clients, wait_genesis
+from tests.tx_generator import actions
+from tests.tx_generator import config as conf
+from tests.tx_generator.models.wallet_api import WalletAPI
+from tests.tx_generator.models.accountant import Accountant
+
+
+def wait_for_next_layer(namespace, timeout):
```

Comment:
this should go into queries if there is no such query yet

Reply:
fixed in 24db1a1e4a26890048ad8a113991652ed0ed3ac3